deploy/olm-catalog/kabanero-operator/0.0.1/kabanero-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,209 @@
+    mediatype: image/png
+  maintainers:
+  - name: Kyle Schlosser
+    email: kschloss@us.ibm.com
+  - name: Tim Kaczynski
+    email: kaczynsk@us.ibm.com
+  - name: Ed Mezarina
+    email: mezarina@us.ibm.com
+  - name: Tim Spewak
+    email: spewak@us.ibm.com
+  - name: Mike Ginnick
+    email: ginnick@us.ibm.com
+  links:
+  - name: Kabanero.io
+    url: https://kabanero.io
+  - name: Kabanero Operator
+    url: https://github.com/kabanero-io/kabanero-operator
+  keywords:
+  - 'Microservice'
+  - 'Knative'
+  - 'Istio'
+  - 'Codewind'
+  - 'Appsody'
+  - 'Tekton'
+  customresourcedefinitions:
+    owned:
+    - description: Kabanero Platform Operator 
+      displayName: Kabanero Operator
+      group: kabanero.io
+      kind: Kabanero
+      name: kabaneros.kabanero.io
+      resources:
+      - kind: ConfigMap
+        name: ""
+        version: v1
+      - kind: Deployment
+        name: ""
+        version: v1
+      - kind: Pod
+        name: ""
+        version: v1
+      - kind: Role
+        name: ""
+        version: v1
+      - kind: RoleBinding
+        name: ""
+        version: v1
+      - kind: ServiceAccount
+        name: ""
+        version: v1
+      version: v1alpha1
+      specDescriptors:
+        - description: Overrides platform component version.
+          displayName: Version
+          path: version
+          x-descriptors:
+            - 'urn:alm:descriptor:com.tectonic.ui:label'
+        - description: Specifies whether Tekton is disabled.
+          displayName: disable Tekton
+          path: tekton.disabled
+          x-descriptors:
+            - 'urn:alm:descriptor:com.tectonic.ui:booleanSwitch'
```

Comment:
If the target environment for this is really OKD 3.x, then we will need to re-consider whether we use x-descriptors or not.  The x-descriptors work in 4.x but not 3.x.  In 4.x if you don't use a spec descriptor, it's hard to describe certain things (like true/false values) because the UI won't display true or false as a value.